qiskit/pulse/commands/set_channel_frequency.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Set channel frequency.
+"""
+from typing import Optional
+
+from qiskit.pulse.channels import PulseChannel
+from .instruction import Instruction
+from .command import Command
+
+
+class SetChannelFrequency(Command):
+    """
+    Set the channel frequency
+    """
+
+    prefix = 'scf'
+
+    def __init__(self, frequency: float, name: Optional[str] = None):
+        """Creates a new set channel frequency pulse.
+
+        Args:
+            frequency: new frequency of the channel in GHz
+            name: name of this set channel frequency command
+        """
+        super().__init__(duration=0)
+        self._frequency = float(frequency)
+        self._name = SetChannelFrequency.create_name(name)
```

Comment:
I think the call to `super` will handle this

Reply:
Thanks for pointing this out.

Reply:
right, I'm not really convinced that `Command`s need a `name` field anyway. I think this is fine, then. Changing `name` behavior is out of scope here.